core/object/message_queue.cpp

```diff
@@ -34,6 +34,7 @@
 #include "core/core_string_names.h"
 #include "core/object/class_db.h"
 #include "core/object/script_language.h"
+#include <cstdio>
```

Comment:
```suggestion

#include <stdio.h>
```

For consistency with the rest of the code.